app/src/main/res/values/styles.xml

```diff
@@ -6,4 +6,24 @@
     <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
     <item name="colorAccent">@color/colorPrimary</item>
   </style>
+
+  <style name="OppiaThemeWithoutActionBar" parent="Theme.AppCompat.Light.NoActionBar">
+    <item name="colorPrimary">@color/colorPrimary</item>
+    <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+    <item name="colorAccent">@color/colorAccent</item>
+  </style>
+
+  <style name="SplashScreenTheme" parent="OppiaThemeWithoutActionBar">
+    <item name="android:windowBackground">@drawable/splash_page</item>
+    <item name="android:windowAnimationStyle">@style/Animation.Activity</item>
+  </style>
+
+  <!-- Base style for animations.  This style specifies no animations. -->
+  <style name="Animation" />
+
+  <!-- Standard animations for a full-screen window or activity. -->
+  <style name="Animation.Activity">
```

Comment:
Maybe instead call this FadeTransitionedActivity or something similar. Not sure if we want this to be the default for all activities, and it will be clearer for each activity that uses this what the style does if we include the fade bit in the name.

Reply:
Changed to FadeTransitionAnimation